src/styles/blocks/catalog.scss

```
@@ -1,17 +1,21 @@
+%speed-duration {
```

```suggestion
%transition-duration {
```
speed-duration? Тривалість швидкості?) 
Але краще винеси в змінну просто 300ms

---

я  просто создал плейсхолдер, который будет глобальным и дал ему такое имя